README.md

```diff
@@ -48,9 +48,10 @@ For the impatient here's the gist of it for Ubuntu:
 
 ```bash
 sudo apt-get install -y software-properties-common
-sudo add-apt-repository -u ppa:bitcoin/bitcoin
 sudo add-apt-repository -u ppa:lightningnetwork/ppa
-sudo apt-get install bitcoind lightningd
+sudo apt-get install lightningd snapd
+sudo snap install bitcoin-core
```

Comment:
I would like to have two sections to handle this information here. Not all people are a fun of snap (like me).

Maybe maintain two versions of this command is better?

Reply:
Maybe it is good to add also this information about the postgress dependencies `sudo apt-get install libpq-dev` in the description https://github.com/ElementsProject/lightning/issues/4539#issue-892725143 

Usually, I need to install in all my machine before being able to use c-lightning

Reply:
In addition, I read your issue on the app, and I think we are a dependencies fork like before ubuntu-20, after ubuntu 20.

In fact, on my docker sandbox, I fixed with this solution (without snap)

```
add-apt-repository ppa:luke-jr/bitcoincore
apt-get update  && apt-get install -y bitcoind
add-apt-repository -u ppa:lightningnetwork/ppa
apt-get update  &&; apt-get install -y lightningd
```

Maybe we need to make a different section for Ubuntu 20?

Reply:
I also needed to install libpq-dev, but only when I installed c-lightning from binary release. With installation of lightningd from apt-get installed it automatically.

Reply:
Ad snap - I just followed official bitcoin core webpage where it links to the snap package as official way how to install if from package on ubuntu https://bitcoincore.org/en/download/

It the luke's ppa works, that is also fine I guess.

Reply:
>I also needed to install libpq-dev, but only when I installed c-lightning from binary release. With installation of lightningd from apt-get installed it automatically.

Yes, sorry, I mean we can add this library in the Library requirement in the same page https://github.com/ElementsProject/lightning/blob/master/doc/INSTALL.md#library-requirements

>Ad snap - I just followed official bitcoin core webpage where it links to the snap package as official way how to install if from package on ubuntu

Yes, the only thing that I don't agree is to remove the bitcoin ppa, in my vision is better to have the choice. Any way it is fine for me

Reply:
@vincenzopalazzo There are multiple ways to run `bitcoind` on Ubuntu. I use the [pre-compiled binaries](https://bitcoincore.org/bin/bitcoin-core-0.21.1/bitcoin-0.21.1-x86_64-linux-gnu.tar.gz) for example. Those can be easily extracted and run as unprivileged user without root access. To install a PPA or snap, root access is needed. The question is how far the rabbit-hole you want our first-time users to go. Personally I would totally love to see everyone compiling from scratch in a musl-libc chroot (on any system, also Ubuntu) but that may be an over-kill for many.

Generally I think it would be much better if exact steps like these would be completely left out of the documentation. It is distribution-specific and not related to C-Lightning. For example, I tried to search [Linux Kernel documentation for a mention of apt](https://www.kernel.org/doc/html/latest/search.html?q=apt&check_keywords=yes&area=default) and did not find anything (which is correct IMHO).

C-Lightning is not Ubuntu-specific despite it is now widely accepted as reproducible-build system. The real reproducible build in my opinion is `-no-pie` statically musl-libc-linked executable which can actually be run on almost _any_ Linux running on the chosen architecture, with no further dependencies.

Excuse me for rant :-)